client/src/store/new-request-module.ts

```diff
@@ -3139,33 +3206,41 @@ export class NewRequestModule extends VuexModule {
       )
     }
   }
+
   @Mutation
   mutateNameAnalysisTimedOut (value: boolean) {
     this.nameAnalysisTimedOut = value
   }
+
   @Mutation
   mutateConditionsModalVisible (value: boolean) {
     this.conditionsModalVisible = value
   }
+
   @Mutation
   clearAssumedNameOriginal () {
     this.assumedNameOriginal = ''
   }
+
   getEntities (category) {
     return this.entityTypesBC.filter(type => type.cat === category)
   }
+
   @Mutation
   mutateUserCancelledAnalysis (value: boolean) {
     this.userCancelledAnalysis = value
   }
+
   @Mutation
   mutateQuickSearch (value: boolean) {
     this.quickSearch = value
   }
+
   @Mutation
   mutateQuickSearchNames (value: Array<object>) {
     this.quickSearchNames = value
   }
+
```

Comment:
You made it to the end! Sorry about all the formatting but the different whitespace styles was driving me nuts.